### PR TITLE
Make rooms / resources automatically reply to invites

### PR DESCRIPTION
**Pull request by georgehrke**
Signed-off-by: Georg Ehrke <developer@georgehrke.com>

### CODE REVIEW

**Comment by georgehrke** [comment]
@rullzer This is finally read for a code-review :)

**Comment by georgehrke** [comment]
The calendar app does not show the response status at the moment:
You have to check the calendar-data of the organiser to check:
8321B77A-3ED5-48CA-B418-5E663B47D335
655B86D1-8B9E-4B6C-AD2B-5F37BA4CF4D4

File: apps/dav/lib/CalDAV/Schedule/Plugin.php [hunk 1]
```diff
@@ -140,4 +298,253 @@ function propFindDefaultCalendarUrl(PropFind $propFind, INode $node) {
...
+		if (isset($vevent->DTEND)) {
+			return $vevent->DTEND;
+		}
+
+		if (isset($vevent->DURATION)) {
+			$isFloating = $vevent->DTSTART->isFloating();
+			/** @var Property\ICalendar\DateTime $end */
+			$end = clone $vevent->DTSTART;
+			$endDateTime = $end->getDateTime();
+			$endDateTime = $endDateTime->add(DateTimeParser::parse($vevent->DURATION->getValue()));
```
Comment (rullzer) [inline apps/dav/lib/CalDAV/Schedule/Plugin.php]: You have to add `use Sabre\VObject\DateTimeParser;` according to phpstorm :)

File: apps/dav/lib/CalDAV/Schedule/Plugin.php [hunk 2]
```diff
@@ -140,4 +298,253 @@ function propFindDefaultCalendarUrl(PropFind $propFind, INode $node) {
...
+		}
+		// RFC 5545 3.2.17: default RSVP is false
+		return false;
+	}
+
+	/**
+	 * @param Property|null $attendee
+	 * @return string
+	 */
+	private function getAttendeePartstat(Property $attendee = null):string {
```
Comment (rullzer) [inline apps/dav/lib/CalDAV/Schedule/Plugin.php]: Not used?

**Review by rullzer — CHANGES_REQUESTED** [review]
Looks good overal

File: apps/dav/lib/CalDAV/Schedule/Plugin.php (hunk 2 above)
Comment (georgehrke) [reply apps/dav/lib/CalDAV/Schedule/Plugin.php]: removed

File: apps/dav/lib/CalDAV/Schedule/Plugin.php (hunk 1 above)
Comment (georgehrke) [reply apps/dav/lib/CalDAV/Schedule/Plugin.php]: Indeed, fixed :)

**Review by rullzer — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**